### PR TITLE
Update Get-SPOGeoAdministrator.md

### PR DESCRIPTION
**Pull request by PowershellScripts**
No other doc seems to use Multi-Geographic.

I wasn't sure about capital letters

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoAdministrator.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ ms.reviewer:
 
 ## SYNOPSIS
 
-This cmdlet returns the SharePoint Online user or security group accounts with global administrative privileges in the current Multi-Geographics tenant.
+This cmdlet returns the SharePoint Online user or security group accounts with global administrative privileges in the current Multi-Geo tenant.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoAdministrator.md:17]:
```suggestion
This cmdlet returns the SharePoint Online user or security group accounts with global administrative privileges in the current multi-geo tenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, kindly commit the suggested change. FYI, Microsoft Multi-Geo is a brand name with initial caps, but I think in this instance, the term is used in its generic form and should not have initial caps. Many thanks.

**Comment by PowershellScripts** [comment]
Thank you, @JohanFreelancer9. The explanation is helpful.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoAdministrator.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ ms.reviewer:
 
 ## SYNOPSIS
 
-This cmdlet returns the SharePoint Online user or security group accounts with global administrative privileges in the current Multi-Geographics tenant.
+This cmdlet returns the SharePoint Online user or security group accounts with global administrative privileges in the current multi-geo tenant.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoAdministrator.md:17]:
```suggestion
This cmdlet returns the SharePoint Online user or security group accounts with Global Admin privileges in the current multi-geo tenant.
```

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by get-itips — APPROVED** [review]
ok after @tseward suggestions

**Comment by yogkumgit** [comment]
@dariomws

**Comment by yogkumgit** [comment]
@PowershellScripts Please implement the suggestions. Thanks

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**